test/docfx.Test/lib/FileGlobTest.cs

```diff
@@ -4,7 +4,8 @@
 using System;
 using System.IO;
 using System.Linq;
-
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
```

Comment:
no change?

Reply:
changed it by testing json dotnet behavior, reverted.

---
In reply to: [200882577](https://github.com/dotnet/docfx/pull/3020#discussion_r200882577) [](ancestors = 200882577)